.github/workflows/ci.yaml

```
@@ -38,10 +38,21 @@ jobs:
     name: build and test (os=${{ matrix.os }})
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Cache ~/.local/bin
+        id: cache-local-bin
+        uses: actions/cache@v4
+        with:
+          path: ~/.local/bin
+          # Cache should be valid for a maximum of 7 days.
+          # <https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy>
+          key: ${{ runner.os }}-local-bin
+
       - name: Install Micromamba
-        run: |
-          curl -fsSL --proto 'https=' https://micro.mamba.pm/install.sh | bash
-          echo ~/.local/bin >> "$GITHUB_PATH"
+        if: steps.cache-local-bin.outputs.cache-hit != 'true'
```

```suggestion
        if: steps.cache-local-bin.outputs.cache-hit != 'true'
```

If we get rid of this we should be able to use the cache as fallback and keep getting new micromamba versions

---

Great suggestion!

---

Done in 15d77232f0cb6e8f64e7d710763daf010b79ebd6.